messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/jms/ExternalJMSQueueDefinition.java

```diff
@@ -46,15 +48,18 @@ public ExternalJMSQueueDefinition(boolean registerRuntimeOnly) {
 
     @Override
     public Collection<AttributeDefinition> getAttributes() {
-        return Collections.singletonList(CommonAttributes.DESTINATION_ENTRIES);
+        return Arrays.asList(CommonAttributes.DESTINATION_ENTRIES, External.ENABLE_AMQ1_PREFIX);
     }
 
     @Override
     public void registerAttributes(ManagementResourceRegistration registry) {
         if (registerRuntimeOnly) {
             registry.registerReadOnlyAttribute(CommonAttributes.DESTINATION_ENTRIES, null);
+            // Should this be read only as entries ?
+            registry.registerReadOnlyAttribute(External.ENABLE_AMQ1_PREFIX, null);
```

Comment:
I wasn't really sure on this "readonly" in the past, is that correct to you @ehsavoie  ?

Reply:
Yes , this is the runtime description of the resource